### ISSUE TITLE
Delete user and Information links not keyboard accessible.

### ISSUE THREAD

**Issue by jcivjan** [instructure/canvas-lms#58]
From the COurse>Settings>Users page, you can't delete individual users or get the information on each one. You should be able to make these keyboard accessible by placing these links offscreen (rather than visibility:hidden) and allowing focus to toggle the same css you use on hover.
